cassis/cas.py

```diff
@@ -612,33 +614,52 @@ def _find_all_fs(self, generate_missing_ids: bool = True) -> Iterable[FeatureStr
             all_fs[fs.xmiID] = fs
 
             t = ts.get_type(fs.type)
+
+            # Arrays contents are handled separately - they only have one "virtual" feature: elements
+            if t.supertypeName == "uima.cas.ArrayBase":
+                if t.name == "uima.cas.FSArray":
```

Comment:
I would swap the condition and do
if t.name != "":
  continue

I hard a hard time to understand the naked continue that is not even in an else block

Reply:
The condition cannot simply be swapped. The continue at the end must always happen, even if the name is not FSArray. I could only swap if I introduced a second continue at the end. IMHO that doesn't improve things.

Reply:
Ok, that makes sense. It is a bit difficult to read imho, but not so relevant to make it clearer